### ISSUE TITLE
please do you have H264 decoder of  cedar  in gstreamer?

### ISSUE THREAD

**Issue by zhang007z** [ebutera/gst-plugin-cedar#2]
recently I learn H264 decoder of  cedar in gstreamer, but I don't H264 decoder of cedar plugin in gstreamer, I find vdpauh264dec plugin in gstreamer.
then I test gstreamer command:
//-----------------------------------------------------------------------------------------//
gst-launch-0.10 -e rtspsrc location=rtsp://192.168.1.6/vs1 protocols=tcp ! rtph264depay ! h264parse ! capsfilter caps="video/x-h264,width=1280,height=720,framerate=(fraction)25/1" ! vdpauh264dec ! ffmpegcolorspace ! cedar_h264enc ! flvmux name=mux ! rtmpsink location="rtmp://127.0.0.1:1935/live/test live=1"  
//------------------------------------------------------------------------------------------//
but returning error:
ERROR: from element /GstPipeline:pipeline0/GstVdpH264Dec:vdph264dec0: Could not open X display with name: (null)
Setting pipeline to NULL ...
 //------------------------------------------------------------------------------------------//
     At above this command uses cedar_h264enc and  vdpauh264dec plugins.it use vdpauh264dec decode H264 video stream ，then it use cedar_h264enc encode raw video stream again. this is my originally opinion.
     According to Allwinner chip A20 introduction,website:http://www.allwinnertech.com/en/clq/processora/A20.html 
this chip support vdpau decoder , this chip is linux-sunxi architecture.this vdpau is "Video Decode and Presentation API for Unix" mean.  if you don't know vdpau .please you use wiki for finding this . I used vdph264dec plugin, but it returned error to said "Could not open X display with name: (null)". I think " vdpauh264dec ! vdpauvideopostprocess ! vdpausink " is necessarily command composited structure. in other words  vdpauh26dec plugin processing of video stream must be received by vdpausink ?? I doubt this problem.  I've been no progress for this test. so I think to ask you that you have cedar H264 decoder for gstreamer plugin ?  I realize this plugin could handle my issue .
you have any idea for me .Thank you for taking the time to answer this question.

**Comment by ebutera** [comment]
yes that pipeline decode the stream with vdpauh264dec and then re-encode it with my gstreamer plugin.

The error string "Could not open X display with name: (null)" suggests that the vdpauh264dec needs the X display server, maybe it was not running when you tried it.
I don't know if it is possibile to compile it disabling the X display server requirement.
